lib/ain-evm/src/evm.rs

```diff
@@ -262,7 +262,7 @@ impl EVMServices {
                 block.header.number, block.header.state_root
             );
 
-            self.block.connect_block(&block)?;
+            self.oracle.connect_block(&block)?;
```

Comment:
The idea behind the rename was because we have been adding functionalities on top of block service such as fee related services and it was acting more as an oracle service for the node at this point. Also it aligns with geth side that implements something similar and named it oracle so i thought that the rename is suitable for us. But switched it back to BlockService in [c1c1278](https://github.com/DeFiCh/ain/pull/2788/commits/c1c1278c7d73e444353e71294315e5ec1d87fdd9)